ps_searchbar.tpl

```diff
@@ -23,12 +23,11 @@
 *  International Registered Trademark & Property of PrestaShop SA
 *}
 
-<div id="search_widget" data-search-controller-url="{$search_controller_url}">
-	<form method="get" action="{$search_controller_url}">
-		<input type="hidden" name="controller" value="search">
-		<input type="text" name="s" value="{$search_string}">
-		<button type="submit">
-			{l s='Search' d='Modules.Searchbar.Shop'}
-		</button>
-	</form>
+<div id="search_widget" class="search-widgets" data-search-controller-url="{$search_controller_url}">
+  <form method="get" action="{$search_controller_url}">
+    <input type="hidden" name="controller" value="search">
```

Comment:
If the form is sent via GET, then why not include this parameter directly in the action? You don't need the hidden input.

Reply:
It seems that parameters are not kept if directly put in the form action.